src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -0,0 +1,436 @@
+subhead: Get information about connected displays and position windows relative to those displays.
+authors:
+  - thomassteiner
+description:
+  The Multi-Screen Window Placement API allows you to enumerate the displays connected to your
+  machine and to place windows on specific screens.
+date: 2020-09-11
+# updated: 2020-09-11
+tags:
+  - blog
+  - capabilities
+hero: hero.jpg
+alt: Simulated trading desk showing multiple fake cryptocurrencies and their price charts.
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/1411878483180650497
+---
+
+{% Aside %}
+  The Multi-Screen Window Placement API is part of the
+  [capabilities project](https://web.dev/fugu-status/) and is currently in
+  development. This post will be updated as the implementation progresses.
+{% endAside %}
+
+The Multi-Screen Window Placement API allows you to enumerate the
+displays connected to your machine and to place windows on specific screens.
+
+### Suggested use cases {: #use-cases }
+
+Examples of sites that may use this API include:
+
+- Multi-window graphics editors à la
+  [Gimp](https://www.gimp.org/release-notes/gimp-2.8.html#single-window-mode) can place various
+  editing tools in accurately positioned windows.
+- Virtual trading desks can show market trends in multiple windows any of which can be viewed in
+  fullscreen mode.
+- Slideshow apps can show speaker notes on the internal primary screen and the presentation on an
+  external projector.
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | [Complete][spec]         |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| **4. Origin trial**                      | **[In progress][ot]**    |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use the Multi-Screen Window Placement API {: #use }
+
+### Enabling via chrome://flags
+
+To experiment with the Multi-Screen Window Placement API locally, without an origin trial token,
+enable the `#enable-experimental-web-platform-features` flag in `chrome://flags`.
+
+### Enabling support during the origin trial phase
+
+Starting in Chrome&nbsp;86, the Multi-Screen Window Placement API will be available as an origin
+trial in Chrome. The origin trial is expected to end in Chrome&nbsp;88 (February&nbsp;24, 2021).
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+### The problem: `Window.open()` is unaware of additional screens
+
+You can open windows with the time-tested
+[`Window.open()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/open) method. While some
```

Comment:
```suggestion
### The problem

The time-tested approach to controlling windows,
[`Window.open()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/open)
is unaware of additional screens
```

Reply:
Maybe more like this. I'm sorry I didn't think of it in the first review.

I think it's good to keep title's short for small screens, when possible.